maps_dashboards/common/index.ts

```diff
@@ -28,6 +28,7 @@ export const APP_PATH = {
 
 export enum DASHBOARDS_MAPS_LAYER_TYPE {
   OPENSEARCH_MAP = 'OpenSearch Map',
+  DOCUMENT_LAYER = 'Document Layer',
```

Comment:
```suggestion
  DOCUMENTS = 'Documents',
```

Reply:
I prefer keep use "document layer" name in code, which is clear to know it means a layer, on UI, change it to Documents.

Reply:
Seems redundant. Your enum already specifies it is layer types.  i believe this will be used by calling as DASHBOARDS_MAPS_LAYER_NAME.DOCUMENT_LAYER , looks very verbose.

Reply:
Updated to DOCUMENTS.